utils/apollo-client.js

```diff
@@ -0,0 +1,18 @@
+const { createHttpLink } = require('apollo-link-http');
```

Comment:
Please, rename it to `apollo-test-client.js` to show that it is explicitly used for the tests

Reply:
fixed